sbnanaobj/StandardRecord/SRFlashMatch.h

```diff
@@ -18,9 +20,16 @@ namespace caf
       virtual ~SRFlashMatch();
 
       bool  present;
-      float score;
       float time;
-      float pe;
+      float charge_q;
+      float light_pe;
+      float score;
+      float scr_y;
+      float scr_z;
+      float scr_rr;
+      float scr_ratio;
+      SRVector3D chargeXYZ;
+      SRVector3D lightXYZ;
```

Comment:
I don't like the `XYZ` suffix. Does something like `chargepos` work?

Reply:
Yeah this is the bit of convention I'm more interested. I see we currently use `dir, end, start, gen, vertex, position...` for describing vector variables.

So maybe we could go a step further and call it `chargeCenter` and `lightCenter`, since they are meant to hold the positions of the weighted centres.